docs/Quick-Start-Guide/Architecture.md

```diff
@@ -315,9 +315,7 @@ backup of entire data for disaster recovery.
 
 Geo-replication uses a master-slave model, whereby replication occurs
 between **Master** - a GlusterFS volume and **Slave** - which can be a
-local directory or a GlusterFS volume. The slave (local directory or
-volume is accessed using SSH tunnel).
-
+local directory or a GlusterFS volume.
```

Comment:
The slave cannot be a non Gluster volume as per the current design. This is outdated doc. Can we remove this as well as a part of this patch?

Reply:
Let me know if the recent change satisfies you or something else is needed.

Reply:
This fixes the outdated doc as of now. This will get outdated once we have path based georep working. So till then we are good.